src/main/java/qna/domain/Answer.java

```diff
@@ -109,10 +92,11 @@ public void setDeleted(boolean deleted) {
     public String toString() {
         return "Answer{" +
                 "id=" + id +
-                ", writerId=" + writerId +
-                ", questionId=" + questionId +
+                ", writerId=" + writer.getId() +
+                ", questionI/d=" + question.getId() +
```

Comment:
```suggestion
                ", questionId=" + question.getId() +
```
오타인 것 같네요 :)

---

toString 에서 객체 자체를 보여줘도 괜찮겠네요, 물론 아시겠지만 연관관계상 순환참조가 발생하지 않도록 유의해주시면 좋을 것 같습니다.